core/src/message/context.rs

```diff
@@ -311,3 +312,95 @@ impl MessageContext {
         (outcome, store)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::MessageContext;
+    use crate::message::context::{ContextSettings, Error};
+
+    #[cfg(debug_assertions)]
```

Comment:
Is that really required? If I'm not mistaken `cargo test` profile inherits `dev` profile so assertions should be on

Reply:
Removed it